setup.py

```diff
@@ -421,39 +421,38 @@ def check_c_extension():
         PACKAGE_DATA['hotdoc.extensions.gi.transition_scripts'] = ['translate_sections.sh']
 
 
-if __name__ == '__main__':
```

Comment:
hm interesting, I guess pip-compile `import`s the module, possibly adding its own hooks to distutils / setuptools prior to that in order to intercept the installed package versions?

Reply:
https://github.com/jazzband/pip-tools/issues/1278

It's a confusing thing to do IMO, not sure how or why some people think this is "best practice". ¯\_(ツ)_/¯
Removing it is certainly not harmful...

Reply:
ack, so as I guessed then. I don't have an opinion on whether it is best practice to do that in a `setup.py` file and I don't really mind either way, but it would have seemed like a good idea for that too to `eval` the script instead of importing it :)